include/SDL3/SDL_hidapi.h

```diff
@@ -235,6 +235,7 @@ extern DECLSPEC SDL_hid_device * SDLCALL SDL_hid_open(unsigned short vendor_id,
  * platform-specific path name can be used (eg: /dev/hidraw0 on Linux).
  *
  * \param path The path name of the device to open
+ * \param bExclusive exclusive opening (windows only?FIXME)
```

Comment:
Yes, `bExclusive` is for windows only. Documentation may be changed
to something like this?
`* \param bExclusive Open device in exclusive mode (Windows only)`